Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```
@@ -18,7 +18,7 @@
 - type: entity
   parent: SpaceCashExpedition
   id: SpaceCashMinimalExpedition
-  name: spesos (5000)
+  name: spesos (5.000)
```

Consistency with en-US cultureinfo.
```suggestion
  name: spesos (5,000)
```